docs/docs/getting-started/04-examples/01-timer-example.md

```diff
@@ -103,50 +138,82 @@ The `events` field of a component associates component predicates with events.
+      predicates:
+        is_timed_out:
+          transition: timer_2
 ```
 
-In this case, after the timer component has been active for 2.5 seconds, it triggers a transition event to `timer_2`.
+In this case, after the timer component has been active for 2 seconds, it triggers a transition event to `timer_2`.
 The `transition` event from `timer` to `timer_2` is a shorthand for unloading the first component and loading the
 second.
 
-The second block describing `timer_2` is nearly identical, as the two timers are intended to have symmetrical behavior.
+The second block describing `timer_2` is nearly identical (apart from a different value for the `timeout` parameter), as
+the two timers are intended to have symmetrical behavior.
+
+The `Timer` components are lifecycle components, they have an internal state which is the elapsed time since the first
+activation. Deactivating a timer saves the elapsed time and stops it from timing out - it is like pausing a stopwatch.
+Activating the timer again is equivalent to unpausing the stopwatch - the elapsed time increases until eventually the
+timeout has been reached. To show this behavior, two trigger buttons can be found in the application. The first button
+pauses the `timer_2` component by deactivating it while the second one un-pauses the timer by activating it.
+
+```yaml
+  buttons:
+    button:
+      display_name: Pause Timer 2
+      on_click:
+        lifecycle:
+          component: timer_2
+          transition: deactivate
+    button_2:
+      display_name: Start Timer 2
+      on_click:
+        lifecycle:
+          component: timer_2
+          transition: activate
+```
 
 ## Run the application
 
 Press the Play button to start the application.
 
 When the application is started, the `timer` component is loaded. It is initially unconfigured, which triggers it
 to be configured. Thereafter, it lands in the inactive lifecycle state, which triggers it to be activated.
-Once activated, the timer starts running. After 2.5 seconds (as specified by the `timeout` parameter),
-the `is_timed_out` predicate goes from false to true. As a result, the `transition` event causes `timer` to be unloaded
-and `timer_2` to be loaded instead. The second timer then goes through the same steps of configuring and activating
-before transitioning back to the first timer.
+Once activated, the timer starts running. After 2 seconds (as specified by the `timeout` parameter), the `is_timed_out`
+predicate goes from false to true. As a result, the `transition` event causes `timer` to be unloaded and `timer_2` to be
+loaded instead. The second timer then goes through the same steps of configuring and activating before transitioning
+back to the first timer.
 
 ![timer example (animated)](./assets/timer-example.gif)
 
-Use the Pause button to keep the application in an idle state; components will remain loaded and active, but predicates
-and events will not be handled until the application is started again.
```

Comment:
I think this is a really important point to keep, to understand what pausing the application means. It already has the potential of being confusing, so if you don't mention it but instead add a manual trigger button that pauses a timer (and not the application itself), it becomes even more confusing. I would keep this line and maybe even extend it a bit further, before introducing the idea of "pausing" (deactivating) just the timer.